src/server/components/files-list.jsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react'
+import styled from 'react-emotion'
+import FileIcon from '@material-ui/icons/Note'
+import MoreHoriz from '@material-ui/icons/MoreHoriz'
+
+import FileActionsMenu from './file-actions-menu'
+import { BodyIconStyle, ActionsContainer } from '../style/icon-styles'
+
+
+const bytesToHumanReadable = (bytes) => {
+  const KB = 1024
+  const MB = KB * 1024
+  const GB = MB * 1024
+  const TB = GB * 1024
+  const round = (n, k = 2) => Math.round(n * ((10 * k) || 1)) / (10 * (k) || 1)
+  if (bytes < 1024) return `${bytes}b`
+  if (bytes < MB) return `${round(bytes / KB, 0)}kb`
+  if (bytes < GB) return `${round(bytes / MB)}mb`
```

Comment:
We might just want to use numeral for this: http://numeraljs.com/

No doubt there are other places where we would want to display large quantities in a more human readable form